guide/content/introduction/configuration.slim

```diff
@@ -15,6 +15,7 @@ markdown:
 pre
   code.highlight.language-ruby
     | Govuk::Components.configure do |conf|
+        conf.brand = "swanky-new-design-system" # or "govuk"
```

Comment:
If this is where people are likely to first encounter this configuration point, it may be worth making it clear the default is "govuk":

```suggestion
        conf.brand = "swanky-new-design-system" # default is "govuk"
```